docs/documentation/index.mdx

```diff
@@ -336,6 +336,41 @@ Check out our video version of this tutorial on YouTube!
 
     </Accordion>
 
+    <Accordion title="iOS setup continuation for physical devices">
```

Comment:
I don't think this should be placed under the accordion tile on setup instruction. I think this should be a separate page that we can link to